### PR TITLE
improve(APIClient): Retain limits for 60 seconds

### PR DESCRIPTION
**Pull request by pxrl**
The new fast relayer is re-querying /limits on every loop, which is unnecessary and increases the chance of hitting a 3s timeout.

### CODE REVIEW

File: src/clients/AcrossAPIClient.ts [hunk 1]
```diff
@@ -108,6 +112,7 @@ export class AcrossApiClient {
       limits: this.limits,
     });
     this.updatedLimits = true;
+    this.updateTimestamp = Math.round(Date.now() / 1000);
```
Comment (nicholaspai) [inline src/clients/AcrossAPIClient.ts:115]: Should this be equal to the value that we compare above?

**Review by nicholaspai — APPROVED** [review]

File: src/clients/AcrossAPIClient.ts (hunk 1 above)
Comment (pxrl) [reply src/clients/AcrossAPIClient.ts]: Updated: https://github.com/across-protocol/relayer-v3/pull/1432/commits/6cf576b43056bb3a6ffb1a29d3f63e728516008a

File: src/clients/AcrossAPIClient.ts [hunk 2]
```diff
@@ -38,8 +40,10 @@ export class AcrossApiClient {
   }
 
   async update(ignoreLimits: boolean): Promise<void> {
-    if (ignoreLimits) {
-      this.logger.debug({ at: "AcrossAPIClient", message: "Skipping querying /limits" });
+    const now = Math.round(Date.now() / 1000);
```
Comment (james-a-morris) [inline src/clients/AcrossAPIClient.ts:43]: Thoughts on using https://github.com/across-protocol/relayer-v2/blob/master/src/utils/TimeUtils.ts#L1-L7

**Review by james-a-morris — APPROVED** [review]
One minor nit

**Review by mrice32 — APPROVED** [review]
LGTM!

File: src/clients/AcrossAPIClient.ts (hunk 2 above)
Comment (pxrl) [reply src/clients/AcrossAPIClient.ts]: Implemented: https://github.com/across-protocol/relayer-v3/pull/1432/commits/9ec0a09af706fd4313d3dce0b202324dbc627a3b

**Pull request merged**